Update launching-a-node.mdx (#654)
o1-labs/docs2: docs/test-world-2/launching-a-node.mdx

:::note

The Testworld Mission 2.0 network: 

- Chain ID: `332c8cc05ba8de9efc23a011f57015d8c9ec96fac81d5d3f7a06969faf4bce92`
- Git SHA-1: `55b78189c46e1811b8bdb78864cfa95409aeb96a`

> The seeds.txt file location: 

Reviewer: ```suggestion
The `seeds.txt file` location: 
```

- Seed List: `https://storage.googleapis.com/seed-lists/testworld-2-0_seeds.txt`

The genesis timestamp is in the future, until this point you will see the following log entry:
```
Time before the chain start time. Waiting $tm_remaining milliseconds before starting participation"
```

Until the chain start time there will not be any gossip, you will see nodes crash with the following message (every ~25 minutes)
```
"[FATAL] *** Mina daemon has been offline for too long ***"
```

> Please configure the node to restart automatically with these instructions [start a mina node with auto-restart flows](/node-operators/connecting-to-the-network#start-a-mina-node-with-auto-restart-flows)

Reviewer: ```suggestion
You must configure the node to restart automatically. Follow the instructions in [Start a mina node with auto-restart flows](/node-operators/connecting-to-the-network#start-a-mina-node-with-auto-restart-flows).
```
